components/more_channels/more_channels.test.tsx

```diff
@@ -302,12 +307,21 @@ describe('components/MoreChannels', () => {
 
         jest.runOnlyPendingTimers();
         expect(wrapper.instance().props.actions.searchMoreChannels).toHaveBeenCalledTimes(1);
-        expect(wrapper.instance().props.actions.searchMoreChannels).toHaveBeenCalledWith('channel', true);
+        expect(wrapper.instance().props.actions.searchMoreChannels).toHaveBeenCalledWith('channel', true, false);
         process.nextTick(() => {
             expect(wrapper.state('search')).toEqual(true);
             expect(wrapper.state('searching')).toEqual(false);
             expect(wrapper.state('searchedChannels')).toEqual([searchResults.data[1]]);
             done();
         });
     });
+
+    test('should hide joined channels from channels props when shouldHideJoinedChannels state is true', () => {
+        const wrapper = shallow<MoreChannels>(
+            <MoreChannels {...baseProps}/>,
+        );
+
+        wrapper.setState({shouldHideJoinedChannels: true});
+        expect(wrapper.instance().activeChannels[0].id).toEqual(baseProps.channels[0].id);
```

Comment:
Can we do a non-existent expectation for the joined channel here to be more true to the test description?

Reply:
Thanks @EricssonLiu I think I got your point correctly and updated the test accordingly. Please let me know what you think